### PR TITLE
I1462 refactor select input fields

### PR DESCRIPTION
**Pull request by Fallen1988**

### CODE REVIEW

File: app/imports/client/react/forms/components/DepartmentsSelectInput.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+const DepartmentsSelectInput = ({
+  organizationId,
+  type,
+  onNewOptionClick,
+  ...props
+}) => (
+  <ApolloSelectInputField
+    {...props}
+    onNewOptionClick={onNewOptionClick}
+    type={type || onNewOptionClick ? 'creatable' : undefined}
```
Comment (Serhiy91) [inline app/imports/client/react/forms/components/DepartmentsSelectInput.js]: It is better to replace and move to destructuring of props of the component
```suggestion
const DepartmentsSelectInput = ({
    organizationId,
    onNewOptionClick,
    type = onNewOptionClick && 'creatable',
    ...props
}) => (
```

Comment (Dartv) [reply app/imports/client/react/forms/components/DepartmentsSelectInput.js]: It is even better to leave this up to user so he should supply `creatable` prop when using `onNewOptionClick`. You can just default this prop to 'creatable'.

File: app/imports/client/react/forms/components/DepartmentsSelectInput.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+      variables: { organizationId },
+    }).then(({ data: { departments: { departments } } }) => ({
+      options: mapEntitiesToOptions(departments),
+    })).catch(swal.error)}
+  />
+);
+
+DepartmentsSelectInput.defaultProps = {
+  multi: true,
+  loadOptionsOnOpen: true,
```
Comment (Dartv) [inline app/imports/client/react/forms/components/DepartmentsSelectInput.js]: You don't need this prop here. `ApolloSelectInputField` already defines this prop

File: app/imports/client/react/forms/components/GoalSelectInput.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+      query: Query.GOAL_LIST,
+      variables: { organizationId },
+    }).then(({ data: { goals: { goals } } }) => ({
+      options: mapEntitiesToOptions(goals),
+    })).catch(swal.error)}
+  />
+);
+
+GoalSelectInput.defaultProps = {
+  loadOptionsOnOpen: true,
```
Comment (Dartv) [inline app/imports/client/react/forms/components/GoalSelectInput.js]: You don't need this prop

File: app/imports/client/react/forms/components/RiskSelectInput.js [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+      query: Query.RISK_LIST,
+      variables: { organizationId },
+    }).then(({ data: { risks: { risks: resultRisks } } }) => ({
+      options: mapRejectedEntitiesToOptions(risks, resultRisks),
+    })).catch(swal.error)}
+  />
+);
+
+RiskSelectInput.defaultProps = {
+  loadOptionsOnOpen: true,
```
Comment (Dartv) [inline app/imports/client/react/forms/components/RiskSelectInput.js]: \-

**Pull request merged**